src/AvionicsCalculations.cpp

```diff
@@ -110,7 +110,8 @@ bool Avionics::calcIncentives() {
   lasController.updateConstants(data.LAS_CONSTANTS);
   LasagnaController::Input lasInput;
   lasInput.h_rel = data.ALTITUDE_BAROMETER;
-  lasInput.h_abs = data.ALTITUDE_CORRECTED;
+  float las_h_abs = data.USE_ALTITUDE_CORRECTED ? data.ALTITUDE_CORRECTED : data.ALTITUDE_BAROMETER;
```

Comment:
Suppose that we switch over at a point when offset is, say 500m. Will the controller panic due to a sharp step or it this handled equivalently to when we change setpoint?

Reply:
Nope, is equivalent to changing setpoint. That's why I introduced `h_rel` and `h_abs`. All things on the controller that relate to velocity are derived from `h_rel`, so a jump in `h_abs` won't do anything.